recipe/meta.yaml

```diff
@@ -104,6 +104,7 @@ outputs:
         - python >=3.7
         - mne-base
         - spyder
+        - bash
```

Comment:
Can we add a `# [win]` selector? Not sure how this will interact with the fact that currently, `mne-installer-menus` is implicitly `noarch` 🤔 Let's see what the linter says?

Reply:
Would require a re-render after adding

Reply:
> Not sure if it's worth it or not...?

It's not an issue if a package is not `noarch`. It basically only increases build time.
So if we agree that we want to have `bash` on Windows, we should do it …
Or do you mean that it's also okay to just ship `bash` on all platforms? Seems the package size is only about 2 MB, which is … really negligible in comparison to the entire installers.

I'll let you decide!

Reply:
> Or do you mean that it's also okay to just ship bash on all platforms? Seems the package size is only about 2 MB, which is … really negligible in comparison to the entire installers.

Yeah that's what I'm thinking. It seems easier than going non-`noarch` and hurting our build time

Reply:
Fine with me!